vegafusion-rt-datafusion/src/data/table.rs

```diff
@@ -14,23 +14,23 @@ use vegafusion_core::error::{Result, ResultWithContext};
 
 #[async_trait]
 pub trait VegaFusionTableUtils {
-    fn from_dataframe_blocking(value: Arc<DataFrame>) -> Result<VegaFusionTable>;
-    async fn from_dataframe(value: Arc<DataFrame>) -> Result<VegaFusionTable>;
+    fn from_dataframe_blocking(value: DataFrame) -> Result<VegaFusionTable>;
+    async fn from_dataframe(value: DataFrame) -> Result<VegaFusionTable>;
     fn pretty_format(&self, max_rows: Option<usize>) -> Result<String>;
     fn to_memtable(&self) -> MemTable;
-    fn to_dataframe(&self) -> Result<Arc<DataFrame>>;
+    async fn to_dataframe(&self) -> Result<DataFrame>;
```

Comment:
DataFusion update replaces convention of use `Arc<DataFrame>` to using `DataFrame` directlu.